discovery_engine_api_tests/tests/front_office/test_personalization.py

```diff
@@ -0,0 +1,42 @@
+import allure
+import pytest
+from handlers.api_handler import ApiHandler
+from utils import test_utils as su
+from utils import assert_utils as au
+from utils import test_utils as tu
+
+
+@allure.suite("Personalization Test Suite")
+class TestPersonalization:
+    api_handler = ApiHandler()
+
+    @allure.severity(allure.severity_level.CRITICAL)
+    def test_personalization(self, ingest_generated_documents_and_interact):
+        user_id = ingest_generated_documents_and_interact
+        request_docs = self.api_handler.get_personalized_docs(user_id)
+        au.assert_status_code_equals(request_docs.status_code, 200)
+
+        documents_response = self.api_handler.deserialize_json(request_docs.text)
+        au.assert_documents_are_valid(documents_response)
+
+    @allure.severity(allure.severity_level.NORMAL)
+    def test_invalid_user_id(self):
+        user_id = su.generate_invalid_id()
+        request_docs = self.api_handler.get_personalized_docs(user_id)
+        au.assert_status_code_equals(request_docs.status_code, 400)
+
+        documents_response = self.api_handler.deserialize_json(request_docs.text)
+        au.assert_strings_equal(documents_response["kind"], "InvalidUserId")
+
+    @allure.severity(allure.severity_level.NORMAL)
+    @pytest.mark.skip(reason="todo: need to be updated")
```

Comment:
❓ does this refer to code in the engine itself, if so is there already a ticket for that?